accessibility/README.md

```diff
@@ -1,16 +1,57 @@
-
 # Accessibility Tools
 
 These scripts are designed to be used to complete an isochrone based accessibility analysis for site selection, coverage, catchment area or microplanning analysis.
 
+Read more about why Mapbox built these tools and how to use them in this blog post
+
 ## Installation
 
-With [conda](https://docs.conda.io/en/latest/) installed, set up an environment with the following
+With [conda][1] installed, set up an environment with the following
 
 `% conda env create -f environment.yml`
 
 `% conda activate geopandas`
 
-## Configuration
+## Usage
+
+To view detailed usage of each script run `python analyze.py --help` or `python isochrone.py --help`
+
+The first step is to run <code>[isochrones.py][2]</code>, a helper tool to generate one [isochrone][3] for each input point, and copy its attributes over to the output shape. Configure the script using the following parameters, which map to parameters [offered by the Isochrone API][4]:
+
+- Input: A geojson or csv files containing points to (*required*)
+- `token`: A Mapbox API token (*required*)
+- `output`: The filename to use for saving the output geometry. Default is inputfile_travelprofile_traveltime.json
+- `minutes`: The travel time in minutes to travel from the point. Default is 30m
+- `profile`: the mode of transport, driving, walking or cycling. Default is driving
+- `generalize`: a tolerance to use for simplifying the output geometries using the Douglas-Peucker algorithm, in meters. Use 0 (the default) to return the full isochrones, and to avoid occasional self-intersecting geometry errors
+- `limit`: If provided, the script will only read in the first *n* features of your input data. Useful for testing
+- `force`: Overwrite any existing output files with the same name. Otherwise, script will error if a file exists
+
+<code>python isochrones.py --help # for detailed usage info
```

Comment:
duplicate of info in line 17

Reply:
do we need to specify python3

Reply:
Depends on how your environment is setup -- but I think for most it defaults to python3 these days, which we specify in the requirements and env